testing.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   rules: {
     'import/no-extraneous-dependencies': 'off',
 
```

Comment:
What's this do?

Reply:
Adapted from the plugin [docs](https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/consistent-test-it.md):

```
/*eslint jest/consistent-test-it: ["error", {"fn": "test", "withinDescribe": "it"}]*/

test('foo'); // valid
describe('foo', function () {
  it('bar'); // valid
});

it('foo'); // invalid
describe('foo', function () {
  test('bar'); // invalid
});
```